modules/core/test/unit/bitgo.ts

```diff
@@ -452,6 +452,29 @@ describe('BitGo Prototype Methods', function() {
       responseData.isValid.should.equal(true);
     }));
 
+    it('should include request body as part of the hmac', co(function* () {
+      const url = 'https://bitgo.fakeurl';
+      const body = { test: 'test' };
+
+      const fixedUnixTime = 1627374646;
+      const originalDateNow = Date.now;
+      Date.now = () => fixedUnixTime;
+
+      try {
+        nock(url)
```

Comment:
Instead of using `nock` to do this test, it would be better to use sinon to mock the `calculateRequestHeaders` method on the bitgo object and check the `text` property on the params object for the correct contents (namely, it should match the string representation of the `body` variable on 457 above)